commit 90d0e628ff11e5030bcc4fc000bca056adda6603
Author: Jia Tan <jiat0218@gmail.com>
Date:   Fri Feb 10 21:38:02 2023 +0800

    liblzma: Add comments for macros in delta.h.
    
    Document LZMA_DELTA_DIST_MIN and LZMA_DELTA_DIST_MAX for completeness
    and to avoid Doxygen warnings.

diff --git a/src/liblzma/api/lzma/delta.h b/src/liblzma/api/lzma/delta.h
index 775647e1..8b8ae5c6 100644
--- a/src/liblzma/api/lzma/delta.h
+++ b/src/liblzma/api/lzma/delta.h
@@ -1,90 +1,98 @@
 /**
  * \file        lzma/delta.h
  * \brief       Delta filter
  * \note        Never include this file directly. Use <lzma.h> instead.
  *
  * See ../lzma.h for information about liblzma as a whole.
  */
 
 /*
  * Author: Lasse Collin
  *
  * This file has been put into the public domain.
  * You can do whatever you want with this file.
  */
 
 #ifndef LZMA_H_INTERNAL
 #	error Never include this file directly. Use <lzma.h> instead.
 #endif
 
 
 /**
  * \brief       Filter ID
  *
  * Filter ID of the Delta filter. This is used as lzma_filter.id.
  */
 #define LZMA_FILTER_DELTA       LZMA_VLI_C(0x03)
 
 
 /**
  * \brief       Type of the delta calculation
  *
  * Currently only byte-wise delta is supported. Other possible types could
  * be, for example, delta of 16/32/64-bit little/big endian integers, but
  * these are not currently planned since byte-wise delta is almost as good.
  */
 typedef enum {
 	LZMA_DELTA_TYPE_BYTE
 } lzma_delta_type;
 
 
 /**
  * \brief       Options for the Delta filter
  *
  * These options are needed by both encoder and decoder.
  */
 typedef struct {
 	/** For now, this must always be LZMA_DELTA_TYPE_BYTE. */
 	lzma_delta_type type;
 
 	/**
 	 * \brief       Delta distance
 	 *
 	 * With the only currently supported type, LZMA_DELTA_TYPE_BYTE,
 	 * the distance is as bytes.
 	 *
 	 * Examples:
 	 *  - 16-bit stereo audio: distance = 4 bytes
 	 *  - 24-bit RGB image data: distance = 3 bytes
 	 */
 	uint32_t dist;
+
+	/**
+	 * \brief       Minimum value for the dist field in lzma_options_delta.
+	 */
 #	define LZMA_DELTA_DIST_MIN 1
+
+	/** 
+	 * \brief       Maximum value for the dist field in lzma_options_delta.
+	 */
 #	define LZMA_DELTA_DIST_MAX 256
 
 	/*
 	 * Reserved space to allow possible future extensions without
 	 * breaking the ABI. You should not touch these, because the names
 	 * of these variables may change. These are and will never be used
 	 * when type is LZMA_DELTA_TYPE_BYTE, so it is safe to leave these
 	 * uninitialized.
 	 */
 
 	/** \private     Reserved field. */
 	uint32_t reserved_int1;
 
 	/** \private     Reserved field. */
 	uint32_t reserved_int2;
 
 	/** \private     Reserved field. */
 	uint32_t reserved_int3;
 
 	/** \private     Reserved field. */
 	uint32_t reserved_int4;
 
 	/** \private     Reserved field. */
 	void *reserved_ptr1;
 
 	/** \private     Reserved field. */
 	void *reserved_ptr2;
 
 } lzma_options_delta;